iModelJsNodeAddon/presentation/UpdateRecordsHandler.cpp

```diff
@@ -55,24 +52,10 @@ struct IModelJsECPresentationUpdateRecordsHandler::FinalReport
         Utf8StringCR GetECDbFileName() const {return m_ecdbFileName;}
     };
 
-    struct PartialHierarchyUpdates : bvector<HierarchyUpdateRecord>
-    {
-    private:
-        uint64_t m_lastUsedTimestamp;
-    public:
-        void push_back(HierarchyUpdateRecord record)
-            {
-            m_lastUsedTimestamp = BeTimeUtilities::GetCurrentTimeAsUnixMillis();
-            bvector<HierarchyUpdateRecord>::push_back(record);
-            }
-        uint64_t GetLastUsedTimestamp() const {return m_lastUsedTimestamp;}
-    };
-
 private:
     static uint64_t const s_partialHierarchyUpdatesLimit = 1000000; // TODO: use 1M for now, need to measure size of update records and tweak this number

```

Comment:
looks like `s_partialHierarchyUpdatesLimit` can be removed now

Reply:
Removed